views/github-configuration.hbs

```diff
@@ -1,116 +1,200 @@
-                          type='submit'
-                        >
-                          {{connectedStatus syncStatus}}
-                        </button>
-                      {{else if (inProgressSync syncStatus)}}
-                        <div class="getConfiguration__loaderContainer">
-                          <div class="getConfiguration__loader"></div>
-                        </div>
-                      {{else}}
-                        <p class="getConfiguration__orgContent__connectedMsg">
-                          {{connectedStatus syncStatus}}
-                        </p>
-                      {{/if}}
+                        Edit
+                      </span>
+                    </a>
+                  {{/if}}
+                </div>
+
+                {{! Connect }}
+                <div class="gitHubConfiguration__orgContent__connect org__cell">
+                  {{#if admin}}
+                    {{#if (isNotConnected syncStatus)}}
+                      <button
+                        class="gitHubConfiguration__orgContent__connectBtn aui-button install-link"
+                        data-installation-id="{{id}}"
+                        type="submit"
+                      >
+                        {{connectedStatus syncStatus}}
+                      </button>
+                    {{else if (inProgressSync syncStatus)}}
+                      <div class="gitHubConfiguration__loaderContainer">
+                        <aui-spinner size="small"></aui-spinner>
+                      </div>
+                    {{else if (failedSync syncStatus)}}
+                      <div class="gitHubConfiguration__orgContent__connectFailed__warningIconContainer">
+                        <span class="gitHubConfiguration__orgContent__connectFailed__warningIcon aui-icon aui-icon-smalll aui-iconfont-warning">
+                          Sync failed warning
+                        </span>
+                      </div>
+                      <button
+                        class="gitHubConfiguration__orgContent__connectFailed aui-button sync-connection-link"
+                        data-jira-host="{{ ../jiraHost }}"
+                        data-installation-id="{{ id }}"
+                      >
+                        Restart backfill
+                      </button>
+                      <input type="hidden" id="_csrf" name="_csrf" value="{{../csrfToken}}">
+                    {{else}}
+                      <p class="gitHubConfiguration__orgContent__connectedMsg">
+                        {{connectedStatus syncStatus}}
+                      </p>
                     {{/if}}
-                  </div>
+                    {{else}}
+                    <button class="gitHubConfiguration__orgContent__connectBtn--disabled aui-button" aria-disabled="true" disabled>
+                      Connect
+                    </button>
+                  {{/if}}
+
+                  {{#if admin}}
```

Comment:
Hmm... I thought this page is available for admins only... What I'm missing?

Reply:
The whole page?? Haha. Nope. See screenshot below for what currently exists is prod. admin just controls whether we can connect or not (like I can't for the Atlassian org).

<img width="923" alt="Screen Shot 2021-10-06 at 6 52 22 pm" src="https://user-images.githubusercontent.com/37155488/136162203-6823ed78-938f-4d98-99d0-65408708f237.png">